Makefile

```diff
@@ -31,9 +31,13 @@ clean:
 distclean: clean
 	find vendor/ -maxdepth 1 -mindepth 1 -not -name vendor.json -exec rm {} -rf \;
 	rm -f manifest/*.yaml
+	rm -f .glide.status
 
-sync:
-	glide install
+sync: .glide.status
+	test -d vendor && \
+	test -f .glide.status && \
+	test '!' .glide.status -ot vendor || \
+		glide install --strip-vendor && touch .glide.status
```

Comment:
Ok, that goes into a way too complex direction for me. Can't we do something simple, like do a .glide.status check when I run "make build", and do an explicit `glide install --strip-vendor`, when I run `make sync`?